utils/actions/markdownHelpers/helper.ts

```diff
@@ -21,12 +22,12 @@ const generalMarkdownHelper = ({
     markdown = `\n #### ${systemResponseName}`;
     markdown += (value?.data || [])
       .map(
-        ({ number, title, link, body }) =>
-          `\n - [#${number} - ${title}](${link}) \n`
+        ({ number, title, link, body, author, created_at }) =>
```

Comment:
Is this related?

Reply:
No, comes from #301 as to not have conflicts, when that one is merged there will be no diff